docs/en/stack/ml/anomaly-detection/ootb-ml-jobs-nginx.asciidoc

```diff
@@ -6,10 +6,89 @@
 ++++
 
 // tag::nginx-jobs[]
-These {anomaly-job} wizards appear in {kib} if you use {filebeat} to ship access 
-logs from your http://nginx.org/[Nginx] HTTP servers to {es} and store it using 
-fields and datatypes from the Elastic Common Schema (ECS). For more details, see
-the {dfeed} and job definitions in
+
+These {anomaly-job} wizards appear in {kib} if you use either {filebeat} or the
+Nginx integration in {fleet} to ship access logs from your
+http://nginx.org/[Nginx] HTTP servers to {es}. The jobs assume that you use
+fields and data types from the Elastic Common Schema (ECS).
+
+[[nginx-access-logs]]
+== Nginx access logs
+
+These {anomaly-jobs} find unusual activity in HTTP access logs.
+
+For more details, see the {dfeed} and job definitions in
+https://github.com/elastic/integrations/blob/{branch}/packages/nginx/kibana/ml_module/nginx-Logs-ml.json[GitHub].
+Note that these jobs are available in {kib} only if data exists that matches the
+{dfeed} query.
+
+low_request_rate_nginx::
+Detects low request rates.
+
+Job details:::
+
+* Analyzes request rates (using the <<ml-count,`low_count` function>>).
+
+Required {beats} or {agent} integrations:::
+
+* Nginx integration 
+
+source_ip_request_rate_nginx::
+Detects unusual source IPs.
+
+Job details:::
+
+* Analyzes request rates (using the <<ml-count,`high_count` function>>)
+relative to all the source IPs (`over_field_name` is `source.address`).
+
+Required {beats} or {agent} integrations:::
+
+* Nginx integration 
+
+source_ip_url_count_nginx::
+Detects unusual source IPs.
+
+Job details:::
+
+* Analyzes distinct counts of URLs (using the
+<<ml-distinct-count,`high_distinct_count` function>> on the `url.original`
+field) relative to all the source IPs (`over_field_name` is `source.address`).
+
+Required {beats} or {agent} integrations:::
+
+* Nginx integration 
+
+status_code_rate_nginx::
+Detects unusual status code rates.
+
+Job details:::
+
+* Analyzes request rates (using the <<ml-count,`count` function>>) split by
+status code (`partition_field_name` is `http.response.status_code`).
+
+Required {beats} or {agent} integrations:::
+
+* Nginx integration 
+
+visitor_rate_nginx::
+Detects unusual visitor rates.
+
+Job details:::
+
+* Analyzes request rates using the <<ml-nonzero-count,`non_zero_count` function>>.
+
+Required {beats} or {agent} integrations:::
+
+* Nginx integration  
+
+[[nginx-access-logs-filebeat]]
+== Nginx access logs ({filebeat})
+
+These legacy {anomly-jobs} find unusual activity in HTTP access logs. For the 
```

Comment:
Typo here - needs to be `{anomaly-jobs}` I guess so that the link is rendered correctly?

Reply:
oof--good catch!  Fixed in cffcbf2ceb0cc0baff4e7c475e844e3c54c7bdd3